integration/clean/clean_dedicated_host.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build clean
+// +build clean
+
+package main
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"log"
+	"time"
+)
+
+// Can't release a host if it was being used within the last 24 hr add 2 hr as a buffer
+var expirationDateDedicatedHost = time.Now().UTC().Add(keepDurationDedicatedHost)
+
+const keepDurationDedicatedHost = -1 * time.Hour * 26
+
+func cleanDedicatedHost() []error {
+	log.Print("Begin to clean EC2 Dedicated Host")
+
+	cxt := context.Background()
+	defaultConfig, err := config.LoadDefaultConfig(cxt)
+	if err != nil {
+		return []error{err}
+	}
+	ec2client := ec2.NewFromConfig(defaultConfig)
+
+	// Get list of dedicated host
+	nameFilter := types.Filter{Name: aws.String("tag:Name"), Values: []string{
+		"IntegrationTestMacDedicatedHost",
```

Comment:
Missed this somehow - I think that this name should be put at the top as a const.